README.md

```diff
@@ -8,26 +8,32 @@ Every great Discord server needs a great bot. So we coded up our own. It does a
+| `cc!mute`             | [user] [reason]                | Admin, Mod     | Mutes a user by assigning them a _Muted_ role (denies message sending and reacting privileges).     |
+| `cc!removeinfraction` | [user] [infractionid]          | Admin, Mod     | Sets a single infraction as invalid.                                                                |
+| `cc!clearmessages`    | [numberofmessages]             | Admin, Mod     | Clears the specified number of messages in the channel where the command is used.                   |
+| `cc!unmute`           | [user]                         | Admin, Mod, SU | Unmutes a user.                                                                                     |
+| `cc!tempmute`         | [user] [lengthoftime] [reason] | Admin, Mod, SU | Temporarily mutes a user for a set time period.                                                     |
+| `cc!kick`             | [user] [reason]                | Admin, Mod, SU | Kicks a user from the server.                                                                       |
+| `cc!warn`             | [user] [reason]                | Admin, Mod, SU | Warns a user of an infraction and logs infraction in db.                                            |
+| `cc!infractions`      | [user]                         | Admin, Mod, SU | Finds user infraction record in db and returns it to channel.                                       |
+| `cc!addnote`          | [user] [note]                  | Admin, Mod, SU | Adds a note to a user.                                                                              |
+| `cc!notes`            | [user]                         | Admin, Mod, SU | Displays all notes that have been added to a user.                                                  |
+| `cc!sendcode`         | [username]                     | Everyone       | Sends a verification code to your Codecademy Discuss email.                                         |
+| `cc!verify`           | [code]                         | Everyone       | Verifies that the code entered is valid and gives you a role for every badge you have on Discourse. |
+| `cc!stats`            | N/A                            | Everyone       | Displays basic server statistics (online members, offline members, total members).                  |
+| `cc!ping`             | N/A                            | Everyone       | Pong!                                                                                               |
+| `cc!helpcenter`       | {plaintext}                    | Everyone       | Provides links to Codecademy's Help Center (either embedded or plaintext).                          |
+| `cc!help`             | {command}                      | Everyone       | Shows information about a given command or lists all commands.                                      |
+
+`[user]` can take either a user mention or user ID.
```

Comment:
Since we're updating this, let's add a note here explaining that arguments in square brackets are mandatory and the ones in curly braces are optional.